mix.exs

```diff
@@ -5,16 +5,16 @@ defmodule Bamboo.Mixfile do
 
   def project do
     [app: :bamboo,
-     version: "0.4.2",
+     version: "0.5.0",
      elixir: "~> 1.2",
      source_url: @project_url,
      homepage_url: @project_url,
      compilers: compilers(Mix.env),
      test_coverage: [tool: ExCoveralls],
      preferred_cli_env: ["coveralls": :test, "coveralls.circle": :test],
      elixirc_paths: elixirc_paths(Mix.env),
-     description: "Straightforward, composable, and adapter based Elixir email library " <>
-     "for people that love piping. Adapters for Mandrill, Sendgrid, Sparkpost, in-memory, and test.",
+     description: "Straightforward, powerful, and adapter based Elixir email library." <>
```

Comment:
Good question! I am not sure, but I believe it does not have a hyphen
> On Apr 29, 2016, at 4:13 PM, Tute Costa <notifications@github.com> wrote:
> 
> In mix.exs <https://github.com/thoughtbot/bamboo/pull/140#discussion_r61638131>:
> 
> >       elixir: "~> 1.2",
> >       source_url: @project_url,
> >       homepage_url: @project_url,
> >       compilers: compilers(Mix.env),
> >       test_coverage: [tool: ExCoveralls],
> >       preferred_cli_env: ["coveralls": :test, "coveralls.circle": :test],
> >       elixirc_paths: elixirc_paths(Mix.env),
> > -     description: "Straightforward, composable, and adapter based Elixir email library " <>
> > -     "for people that love piping. Adapters for Mandrill, Sendgrid, Sparkpost, in-memory, and test.",
> > +     description: "Straightforward, powerful, and adapter based Elixir email library." <>
> Is adapter-based a compound word (does it need a hyphen)?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/thoughtbot/bamboo/pull/140/files/aeb397335ce0f38bad3ca76c2fdc7551d812cf69#r61638131>